client/blocks/nps-survey/index.jsx

```diff
@@ -73,9 +78,6 @@ class NpsSurvey extends Component {
 			'is-submitted': this.props.hasAnswered,
 		} );
 
-		analytics.mc.bumpStat( 'calypso_nps_survey', 'survey_displayed' );
```

Comment:
Calling analytics here causes two issues:

1. A warning to be output on dev console every time the component renders...

```
Warning: setState(...): Cannot update during an existing state transition (such as within `render` or another component's constructor). Render methods should be a pure function of props and state; constructor side-effects are an anti-pattern, but can be moved to `componentWillMount`.
```

2. The analytics to be called every time the component renders, which includes every time a new radio button is selected.

The correct place for such a call is in `componentWillMount` -- that way, it will just be called once, right before the component mounts (and it displayed).